cognite/client/_api/annotations.py

```diff
@@ -0,0 +1,138 @@
+
+        Returns:
+            Union[Annotation, AnnotationList]: created annotation(s)
+        """
+        assert_type(annotations, "annotations", [Annotation, list])
+        return self._create_multiple(resource_path=self._RESOURCE_PATH + "/", items=annotations)
+
+    def suggest(self, annotations: Union[Annotation, List[Annotation]]) -> Union[Annotation, AnnotationList]:
+        """Suggest annotations
+
+        Args:
+            annotations (Union[Annotation, List[Annotation]]): annotation(s) to suggest. They must have status set to "suggested".
+
+        Returns:
+            Union[Annotation, AnnotationList]: suggested annotation(s)
+        """
+        assert_type(annotations, "annotations", [Annotation, list])
+        # Deal with status fields in both cases: Single item and list of items
+        items = (
+            [AnnotationsAPI._sanitize_suggest_item(ann) for ann in annotations]
+            if isinstance(annotations, list)
+            else AnnotationsAPI._sanitize_suggest_item(annotations)
+        )
+        return self._create_multiple(resource_path=self._RESOURCE_PATH + "/suggest", items=items)
+
+    @staticmethod
+    def _sanitize_suggest_item(annotation: Annotation) -> Dict[str, any]:
+        # Check that status is set to suggested if it is set and afterwards remove it
+        item = annotation.dump(camel_case=True) if isinstance(annotation, Annotation) else deepcopy(annotation)
+        if "status" in item:
+            if item["status"] != "suggested":
+                raise ValueError("status field for Annotation suggestions must be set to 'suggested'")
+            del item["status"]
+        return item
+
+    def list(self, filter: Union[AnnotationFilter, Dict], limit: int = 25) -> AnnotationList:
+        """List annotations.
+
+        Args:
+            limit (int): Maximum number of annotations to return. Defaults to 25.
+            filter (AnnotationFilter, optional): Return annotations with parameter values that matches what is specified. Note that annotated_resource_type and annotated_resource_ids are always required.
+
+        Returns:
+            AnnotationList: list of annotations
+        """
+        assert_type(limit, "limit", [int], allow_none=False)
+        assert_type(filter, "filter", [AnnotationFilter, dict], allow_none=False)
+
+        if isinstance(filter, AnnotationFilter):
+            filter = filter.dump(camel_case=True)
+
+        elif isinstance(filter, dict):
+            filter = {to_camel_case(k): v for k, v in filter.items()}
+
+        if "annotatedResourceIds" in filter:
+            filter["annotatedResourceIds"] = [
+                {to_camel_case(k): v for k, v in f.items()} for f in filter["annotatedResourceIds"]
+            ]
+
+        if "linkedResourceIds" in filter:
+            filter["linkedResourceIds"] = [
+                {to_camel_case(k): v for k, v in f.items()} for f in filter["linkedResourceIds"]
+            ]
```

Comment:
Can be removed as "linkedResourceIds" should never appear in the filter anymore

Reply:
[5cdf073](https://github.com/cognitedata/cognite-sdk-python/pull/928/commits/5cdf073ea643d7be6d7bfbe83874307d072be959)